dizoo/bsuite/envs/test_bsuite_env.py

```diff
@@ -0,0 +1,57 @@
+from time import time
+import pytest
+import numpy as np
+from easydict import EasyDict
+from dizoo.bsuite.envs import BSuiteEnv
+
+
+@pytest.mark.unittest
```

Comment:
we use `envtest` mark in test related to environment

Reply:
Received, I will modify accordingly. However I noticed that also other envs such as Atari use the `unittest ` mark, should it be changed?

Reply:
Yes, we will fix it in the following version.